src/main/java/mocha/game/world/Location.java

```diff
@@ -32,8 +39,14 @@ public void addY(double y) {
   @Override
   public String toString() {
     return "{" +
-        "  \"x\": " + x + "," +
-        "  \"y\": " + y + "" +
-        "}";
+      "  \"x\": " + x + "," +
+      "  \"y\": " + y + "" +
+      "}";
+  }
+
+  @Override
+  public boolean equals(Object myObject) {
```

Comment:
When you override equals(), you should also override hashCode()
https://www.javaworld.com/article/2074996/hashcode-and-equals-method-in-java-object---a-pragmatic-concept.html

As well, we need to enhance this to take into account the type of the object we're comparing with.
For example, if we are asking for the equality between new Location().equals(new Object()).
In this case, the cast won't succeed and we'll have issues.

Normally we don't have to cover every edge case, but equals()/hashCode() are used together by Java natively and that introduces risk.

We can deal with it later.